pages/common/az-config.md

```diff
@@ -0,0 +1,21 @@
+# az config
+
+> Manage Azure CLI configuration.
+> Part of `azure-cli`.
+> More information: <https://docs.microsoft.com/cli/azure/config>.
+
+- Print all configurations:
+
+`az config get`
+
+- Get configurations for a section:
```

Comment:
```suggestion
- Print configurations for a section:
```

I assume you didn't see the small comment, no harm done.

Reply:
Sorry! I read it but I forgot to change it... 😢
Committing the change now. Thanks for seeing it.

Reply:
If you ever want a change, you can use VSCode in the browser by pressing `.` at any time. However the interface to me isn't as easy as plain git

Reply:
Great tip! I didn't know about it...